games-action/minetest_game/minetest_game-5.0.1.ebuild

```diff
@@ -0,0 +1,23 @@
+# Copyright 1999-2019 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+DESCRIPTION="The main game for the Minetest game engine"
+HOMEPAGE="https://github.com/minetest/minetest_game"
+SRC_URI="https://github.com/minetest/${PN}/archive/${PV}.tar.gz -> ${P}.tar.gz"
+
+LICENSE="GPL-2 CC-BY-SA-3.0"
```

Comment:
I wanted to say this is such a nice simple ebuild but actually, it isn't. The files apparently have more licenses than you've noted here. Please look through license information in all the installed files, and assemble a complete list.